docs/en/interfaces/predefined_http.md

```diff
@@ -0,0 +1,216 @@
+# Predefined HTTP Interface {#predefined_http_interface}
+
+ClickHouse already supports specific queries through the [HTTP interface](../../interfaces/HTTP/http.md) . For example, you can write data to a table as follows:
+ 
+```bash
+$ echo '(4),(5),(6)' | curl 'http://localhost:8123/?query=INSERT%20INTO%20t%20VALUES' --data-binary @-
+```
+
+Now ClickHouse supports Predefined HTTP Interface which can help you more easy integration with third party tools. For example, the following example describes how to by Prometheus with Predefined HTTP Interface integration.
+
+> Example:
+> 
+> * First, add these configurations to the configuration file:
+> 
+> ``` xml
+> <http_handlers>
+>	<predefine_query_handler>
+>	    <url>/metrics</url>
+>	    <method>GET</method>
+>	    <queries>
+>	        <query>SELECT * FROM system.metrics LIMIT 5 FORMAT Template SETTINGS format_template_resultset = 'prometheus_template_output_format_resultset', format_template_row = 'prometheus_template_output_format_row', format_template_rows_between_delimiter = '\n'</query>
+>	    </queries>
+>	</predefine_query_handler>
+> </http_handlers>
+> ```
+> 
+> * You can now request the url directly for data in the Prometheus format:
```

Comment:
* link to format description and/or to Prometheus itself
* we should explain in the very beginning that this example is about exporting metrics in Prometheus format, not in the middle of it